CRM/Core/BAO/FinancialTrxn.php

```diff
@@ -494,9 +478,6 @@ public static function getPartialPaymentWithType($entityId, $entityName = 'parti
         elseif ($paymentVal > 0) {
           $value['amount_owed'] = $paymentVal;
         }
-        elseif ($lineItemTotal == $ftTotalAmt) {
-          $value['full_paid'] = $ftTotalAmt;
-        }
```

Comment:
lol and I didn't relaised it's not used anwhere in Civi but still we are having a separate attribute.
```
$ grep -irn "full_paid" CRM/
CRM//Core/BAO/FinancialTrxn.php:498:          $value['full_paid'] = $ftTotalAmt;
```
Agree with this change